profile/README.md

```diff
@@ -12,9 +12,12 @@
         <a href="https://medium.com/@process-analytics" style="margin-right: 15px">
           <img alt="medium" src="https://raw.githubusercontent.com/process-analytics/.github/main/profile/logo/medium.svg" height="35px"> 
         </a> 
-        <a href="https://dev.to/process-analytics">
+        <a href="https://dev.to/process-analytics" style="margin-right: 15px">
           <img alt="dev.to" src="https://raw.githubusercontent.com/process-analytics/.github/main/profile/logo/dev_to.svg" height="35px"> 
         </a> 
+        <a href="https://discord.gg/GKhcFpZKkm">
+          <img alt="discord" src="./logo/discord.svg" height="35px"> 
```

Comment:
⚠️ remember #6, that's why we are using absolute urls for logo and images. See also #7